### PR TITLE
Better Mongo support inc mappings and Managers

### PR DESCRIPTION
**Pull request by dao**

### CODE REVIEW

File: Document/AttendeeManager.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+class AttendeeManager extends BaseAttendeeManager
+{
+
+    /**
+     * @var DocumentManager
+     */
+    protected $em;
+
+    /**
+     * @var EntityRepository
```
Comment (robzienert) [inline Document/AttendeeManager.php]: I assume you mean `DocumentRepository`?

File: Document/CalendarManager.php [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+class CalendarManager extends BaseCalendarManager
+{
+
+    /**
+     * @var EntityManager
+     */
+    protected $em;
+
+    /**
+     * @var EntityRepository
```
Comment (robzienert) [inline Document/CalendarManager.php]: Same goes here, DocumentRepository?

File: Resources/config/doctrine/Calendar.mongodb.xml [hunk 3]
```diff
@@ -4,12 +4,13 @@
                         xsi:schemaLocation="http://doctrine-project.org/schemas/odm/doctrine-mongo-mapping
                         http://doctrine-project.org/schemas/odm/doctrine-mongo-mapping.xsd">
 
-    <document name="Rizza\CalendarBundle\Document\Calendar" table="calendar_calendar">
-        <field name="id" id="true"/>
+    <mapped-superclass name="Rizza\CalendarBundle\Document\Calendar" table="calendar_calendar">
+<!-- <field name="id" id="true"/>-->
```
Comment (robzienert) [inline Resources/config/doctrine/Calendar.mongodb.xml]: Just delete it if it's not being used.

**Comment by robzienert** [comment]
:bear: 

Trusting that this works, I haven't used Symfony in almost a year. Thanks for this!

**Comment by dao** [comment]
It does work - I'm able to create a calendar, add events, and add myself as an attendee. One does need to extend the bundle to add documents extending the mapped superclasses to define relations to e.g. your own user class.

**Comment by permanaj** [comment]
Hi @dao , I'm using Symfony 2.2 and I extend the bundle. When I try to run command like `router:debug` it return error `[Doctrine\ODM\MongoDB\Mapping\MappingException]                              
  Class Rizza\CalendarBundle\Document\Event is not a valid document or mapped  
   super class.` 
What did I missing?

**Pull request merged**